src/inspector/inspectorhandler.cpp

```diff
@@ -380,8 +380,6 @@ static QList<InspectorCommand> s_commands{
 
                        QPointF pointF = item->mapToScene(QPoint(0, 0));
                        QPoint point = pointF.toPoint();
-                       point.rx() += item->width() / 2;
-                       point.ry() += item->height() / 2;
```

Comment:
why was this removed?

Reply:
I left a lareg message on the commit, tl;dr; it's a workaround for tests to pass. See https://github.com/mozilla-mobile/mozilla-vpn-client/pull/8139/commits/6e546828ca46a3ada5860c1b3814834b6f80dc8b